lib/tools/system-calls.js

```diff
@@ -419,15 +420,24 @@ systemCallMethods.checkAvdExist = async function (avdName) {
   } catch (e) {
     let unknownOptionError = new RegExp("unknown option: -list-avds", "i").test(e.stderr);
     if (!unknownOptionError) {
-      log.errorAndThrow(`Error executing checkAvdExist. Original error: '${e.message}'; ` +
-                        `Stderr: '${(e.stderr || '').trim()}'; Code: '${e.code}'`);
+      log.errorAndThrow(`Error executing checkAvdExist. Original error: '${e.message}'; Stderr: '${(e.stderr || '').trim()}'; Code: '${e.code}'`);
+    }
 
+    let version = await getSdkToolsVersion();
```

Comment:
Also, there should be some verification for the case we cannot parse SDK version number